srtp/srtp.c

```diff
@@ -3358,6 +3359,10 @@ update_template_streams(srtp_t session,
     }
   }
 
+  /* Copy master key copy as provided by the application */
```

Comment:
This would replicate the SRTP master key, but if PRIME is used, there are additional checks.  For EKT, we also have to ensure we copy over the SPI value, etc. Do you want to tackle that or would you like me to do it?